docs/reference/advanced_configuration.rst

```diff
@@ -183,5 +183,5 @@ Full configuration options:
     jms_serializer:
         metadata:
             directories:
-                - { name: 'sonata_datagrid', path: '%kernel.project_dir%/vendor/sonata-project/datagrid-bundle/src/Resources/config/serializer', namespace_prefix: 'Sonata\DatagridBundle' }
+                - { name: 'sonata_admin', path: '%kernel.project_dir%/vendor/sonata-project/admin-bundle/src/Resources/config/serializer', namespace_prefix: 'Sonata\AdminBundle' }
```

Comment:
This should be fixed, the thing is where should this serializer config should be? on Admin or on persistence?

Ref: https://github.com/sonata-project/SonataDatagridBundle/blob/3.x/src/Resources/config/serializer/Pager.BasePager.xml

(IMO it would be better if it is placed on the admin-bundle)

wdyt @VincentLanglet

Reply:
What is this file used for ?

Reply:
Not really sure TBH because I don't use the APIs from Sonata. I think it is used for the FOSRest, in order to return the Pager and have it serialized to json/xml.

Reply:
Isn't better for bundle like mediaBundle to have his own serializer config in case you want to serialize the Pager differently ?

Reply:
With that approach we will end with N Pager serializer configurations, that are all the same, because the fields on the Pager class are always the same, not sure if I understood you correctly.

Reply:
Yes kinda.

But on the otherside why would we need to provide a serializer configuration in the SonataAdmin, not use by SonataAdmin and only for the Pager. A bundle could start to serialize the datagrid. A user could start to serialize the ProxyQuery. Isn't it ?

We can add it to SonataAdmin for simplicity for sure. But if we do, it would require some tests functional tests too.